spec/game_spec.rb

```diff
@@ -0,0 +1,150 @@
+require_relative '../lib/game'
+
+describe Game do
+  describe '#add' do
+    subject(:game) { described_class.new }
+
+    it 'adds the coin to the bottom of the column' do
+      column = 3
```

Comment:
You have the same value of `column` and `symbol` between all of these tests. You can move the out of the tests to your setup, and add
```
  let(:symbol) { "X" }
  let(:column) { 3 } 
```
In the `describe "#add"`